model-optimizer/extensions/middle/reverse_tensor_iterator.py

```diff
@@ -109,6 +126,9 @@ def replace_pattern(self, graph: Graph, match: dict):
                         port['start'] = None
                         port['end'] = None
```

Comment:
I think `end` should be set individually if we have a constant in `seq_len`  not equal to the whole sequence size.

Reply:
If seq_len is not equal then transformation will not be applied. We check that seq_len = seq_lengths is is_fusable_sequence (in dynamic case we check that subgraph takes value from shape and set to input)